Assets/Scripts/Text/ConversationTextManager.cs

```diff
@@ -108,6 +120,13 @@ public void Initialize(string fileName)
         DisplayText();
     }
 
+    private void LoadJson(string fileName)

```

Comment:
```suggestion
    private TalkData LoadJson(string fileName)
```
にするか、そもそもInitializeFromJsonに統合するか